azure-iot-hub-devicesdk/samples/send_to_output.py

```diff
@@ -22,7 +22,7 @@
 module_client.connect()
 
 # send 5 messages with a 1 second pause between each message
-for i in range(0, 5):
+for i in range(1, 6):
```

Comment:
>(1, 6): [](start = 14, length = 7)

this seems unrelated to the main objective of the PR (not a blocker, just making sure this is intentional)

Reply:
"Made minor sample adjustments to align output with similar samples."

This sample was the one that was causing problems, and in taking a close look at it, I realized the output wasn't alligned with most other samples, so I fixed it too.

---
In reply to: [268366269](https://github.com/Azure/azure-iot-sdk-python-preview/pull/69#discussion_r268366269) [](ancestors = 268366269)